Dockerfile

```diff
@@ -1,15 +1,22 @@
-FROM node:10
+# Use alpine node version
+FROM node:10.15.3-alpine
 
-# Create app directory
-WORKDIR /usr/local
+# Create working directory and set node user as owner
+WORKDIR /usr/src/app
+RUN chown node:node /usr/src/app
+
+# Switch to node user
+USER node
 
 # Install app dependencies
 # A wildcard is used to ensure both package.json AND package-lock.json are copied
 # where available (npm@5+)
-COPY package*.json ./
-
+COPY --chown=node:node package*.json ./
 RUN npm install
+# If you are building your code for production
+# RUN npm ci --only=production
 
-RUN selenium-standalone start
+# Bundle app source setting node user as owner
+COPY --chown=node:node . .
 
-EXPOSE 4444
+CMD yarn run wdio
```

Comment:
Selenium now runs in a separate container so no need to expose any ports from the acceptance test container.